### DISCUSSION TITLE
<title>Reporting mechanism for security issues?</title>

### DISCUSSION THREAD

**Discussion by forrie** [community/community#110776] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Instead of burying real, actual support behind a paywall -- would it make sense to have a link to report actual, real security issues that take place on this site?  For example, this pirate site is now spamming your service:

https://github.com/orgs/ACTMovies/discussions/179

I'm really tired of seeing these things happen, only to be confronted with a lot of messy content and navigation that makes no sense to me, when this needs to be reported.

Kindly consider providing this essential public function.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @forrie 👋🏼  We really appreciate you flagging this and feedback on our reporting process. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
